datastore/google/cloud/datastore/entity.py

```diff
@@ -181,6 +181,19 @@ def kind(self):
         if self.key:
             return self.key.kind
 
+    @property
+    def id(self):
+        """Get the id of the current entity.
+
+        .. note::
+          This relies entirely on the :class:`gcloud.datastore.key.Key`
```

Comment:
Note that this package has been `google.cloud.datastore` (not `gcloud.datastore`) for over a year